.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,16 +5,19 @@
 - [ ] The top-level JSON objects are sorted alphabetically
 - [ ] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update
 
-#### for websites-with-shared-credential-backends.json
-- [ ] There's evidence the domains are related (SSL certificates, DNS entries, valid links between sites, legal documents etc.)
-- [ ] The new group serves login pages on each of the included domains, and those login page accept accounts from the others. (For example, we don't associate `google.co.il` to `google.com`, because `google.co.il` redirects to `accounts.google.com` for authentication.)
-
-#### for change-password-URLs.json
-- [ ] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)
-- [ ] The URL either makes the experience better or no worse than being directed to just the domain in a non-logged-in state
-
 #### for password-rules.json
 - [ ] The given rule isn't particularly standard and obvious for password managers
 - [ ] Generated passwords have been tested from this rule using the [Password Rules Validation Tool](https://developer.apple.com/password-rules/)
 - [ ] Information has been included about the website's requirements (eg. screenshots, error messages, steps during experimentation, etc.)
 - [ ] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
+
+#### for change-password-URLs.json
+- [ ] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)
+- [ ] The URL either makes the experience better or no worse than being directed to just the domain in a non-logged-in state
+
+#### for shared-credential.json
+- [ ] There's evidence the domains are currently related (SSL certificates, DNS entries, valid links between sites, legal documents etc.)
+- [ ] The new group serves login pages on each of the included domains, and those login page accept accounts from the others. (For example, we don't associate `google.co.il` to `google.com`, because `google.co.il` redirects to `accounts.google.com` for authentication.)
```

Comment:
We should probably mention redirects here as well :slightly_smiling_face:

Reply:
@Cldfire what do you mean by redirects?

Reply:
@erynofwales one of the main motivations behind the updated format is first-class support for declaring redirects 🙂 such as `discordapp.com` -> `discord.com`.

The concept generalizes beyond that, but explaining it terms of its most common form (redirects) feels like a good fit for the pull request template.